crypto/rand/drbg_ctr.c

```diff
@@ -422,13 +422,19 @@ int drbg_ctr_init(RAND_DRBG *drbg)
         drbg->max_perslen = DRBG_MAX_LENGTH;
         drbg->max_adinlen = DRBG_MAX_LENGTH;
     } else {
+#ifdef FIPS_MODE
+        RANDerr(RAND_F_DRBG_CTR_INIT,
+                RAND_R_DERIVATION_FUNCTION_MANDATORY_FOR_FIPS);
```

Comment:
It is ok to require a derivation function in FIPS mode, but not because it is "not possible", but rather because we did not implement it. So I'd rather prefer a more neutral formulation of the commit (and pr) title, e.g.

```
Reseeding without derivation function is not supported in FIPS mode.
```

Reply:
Changed the commit and subject.